solr/solrj/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -128,6 +128,7 @@
 
   public static final String CONFIGS_ZKNODE = "/configs";
   public final static String CONFIGNAME_PROP = "configName";
+  public final static String COLLECTION_CONFIG_PROP = "collection.configName";
```

Comment:
Probably doesn't go here because I think it's specific to the HTTP API layer.  This class is too internal to declare such a name.